src/views/dashboard/New.vue

```diff
@@ -81,6 +81,10 @@ const onCreate = () => {
       states.entity === d.entity &&
       states.selectedLayer === d.layer
   );
+  if (!states.name) {
```

Comment:
I noticed this would use `trim`, does this cover the name only includes whitespace?

Reply:
> does this cover the name only includes whitespace?

Yes.